### PR TITLE
Modify the sample code to es2015 syntax in README.md

### PR DESCRIPTION
**Pull request by abouthiroppy**
Changes proposed:
We use `const` in README.md's sample.

Upgrade Path (for changed or removed APIs):

Acceptance Checklist:
- [ ] All commits have been squashed to one.
- [ ] The commit message follows the guidelines in `CONTRIBUTING.md`.
- [ ] Documentation (README.md) and examples have been updated as needed.
- [ ] If this is a code change, a spec testing the functionality has been added.
- [ ] If the commit message has [changed] or [removed], there is an upgrade path above.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,9 +122,9 @@ You can use this to remove scrolling on the the body while the modal is open.
 Inside an app:
 
 ```js
-var React = require('react');
-var ReactDOM = require('react-dom');
-var Modal = require('react-modal');
+const React = require('react');
```
Comment (claydiffrient) [inline README.md]: Should we go ahead and do `import React from 'react';` here?

File: README.md [hunk 2]
```diff
@@ -155,7 +155,7 @@ const customStyles = {
 };
 
 
-var App = React.createClass({
+const App = React.createClass({
```
Comment (claydiffrient) [inline README.md]: Perhaps convert this over to ES2015 class syntax as well?

Comment (abouthiroppy) [inline README.md]: OK, so I'll change es2015 syntax :)

**Comment by abouthiroppy** [comment]
@claydiffrient Updated.

**Pull request merged**